capa/features/extractors/dnfile/extractor.py

```diff
@@ -68,9 +68,10 @@ def get_type(self, token: int) -> Optional[Union[DnType, DnUnmanagedMethod]]:
 
 
 class DnfileFeatureExtractor(FeatureExtractor):
-    def __init__(self, path: str):
+    def __init__(self, path: str, len: int):
         super().__init__()
         self.pe: dnfile.dnPE = dnfile.dnPE(path)
+        self.len = len
```

Comment:
lets call this `min_str_len` since its not obvious from the name `len` that its used by the string extractor.

Reply:
Changed all names to `min_len` as stated in a previous comment.